lib/api/client-commands/network/mockResponse.js

```diff
@@ -20,7 +20,8 @@ const {Logger} = require('../../utils');
  *    });
  *  });
  *
- * @method mockNetworkResponse
+ * @method network.mockResponse
+ * @syntax .network.mockResponse(urlToIntercept, {status, headers, body}, [callback])
  * @syntax .mockNetworkResponse(urlToIntercept, {status, headers, body}, [callback])
```

Comment:
Can we move `.mockNetworkResponse` above `.network.mockResponse`, so that the main command appears first in docs page? Same for `.setNetworkConditions`.